### ISSUE TITLE
loss funciton

### ISSUE THREAD

**Issue by Sean0719** [pytorch/pytorch#50233]
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)

Hello, 

Is there exist one loss function with the similar function of keras.losses.SparseCategoricalCrossentropy. Thanks.

**Comment by ngimel** [comment]
Please ask questions on discuss.pytorch.org.
